packages/@okta/i18n/src/properties/login.properties

```diff
@@ -303,14 +303,14 @@ email.code.label = Verification code
 email.code.not.received = Haven\'t received an email?
 email.enroll.title = Set up Email Authentication
 email.enroll.description = Send a verification code to your registered email.
+email.enroll.enterCode = Enter code
```

Comment:
Interestingly, this is the **5th** entry to translate the text "Enter code". I think this is OK for now, but wanted to bring this to @nikhilvenkatraman-okta's attention as we're starting to see more duplicates in this file.

Reply:
yeah, ok for now. We are supporting customization for each of these strings by doing this, but I agree that this is creating some duplicate entries which is not ideal.